app/Main.hs

```diff
@@ -214,9 +220,9 @@ runZeroMQConnection options config s_interrupted verbose' = do
                         ZMQ.connect failureSocket $ printf "tcp://%s:%d" failureHost failurePort
                         verbose' $ printf "Pushing failed parses on tcp://%s:%d" failureHost failurePort
 
-                        let normalisationConduit = case oJsonInput options of
-                                                        True  -> CB.lines $= C.map (normaliseJsonInput fs)
-                                                        False -> DCT.decode DCT.utf8 $= DCT.lines $= C.map (normaliseText fs)
+                        let normalisationConduit = if oJsonInput options
+                                                        then CB.lines $= C.map (normaliseJsonInput fs)
+                                                        else DCT.decode DCT.utf8 $= DCT.lines $= C.map (normaliseText fs)
```

Comment:
thrice ;-)

Reply:
Idem. I'm trying to resolve this, but so far, no way out.

Reply:
Fixed.